src/app/components/ThemeProviderCSSModules/fontFamilies.module.css

```diff
@@ -0,0 +1,20 @@
+:root {
+  --font-family-apple-sd-gothic-pro: "Apple SD Gothic Neo", AppleGothic, "Malgun Gothic", Dotum, "Noto Sans CJK KR", sans-serif;
+  --font-family-arial: Arial, Verdana, Geneva, Helvetica, sans-serif;
+  --font-family-helmet: Helmet, Freesans, Helvetica, Arial, sans-serif;
+  --font-family-helvetica-chinese: Helvetica, Arial, STHeiti, 华文黑体, "Microsoft YaHei", 微软雅黑, SimSun, 宋体;
+  --font-family-hiragino-kagu-gothic-pro: "Hiragino Kaku Gothic Pro", "ﾋﾗギﾉ角ゴPro W3", "MS PGothic", "MS UI Gothic", Helvetica, Arial, sans-serif;
+  --font-family-noto-sans-telugu: "Noto Sans Telugu", "Hind Guntur", Gautami, "Kohinoor Telugu", sans-serif;
+  --font-family-noto-sans-gujarati: "Noto Sans Gujarati", Rasa, "Hind vadodara", Shruti, Gautami, "Gujarati Sangam MN", "Gujarati MT", sans-serif;
+  --font-family-noto-sans-bengali: "Noto Serif Bengali", Helmet, Freesans, Helvetica, Arial, sans-serif;
+  --font-family-noto-sans-devanagari: "Noto Sans Devanagari", Hind, "Mukta Vaani", Gautami, "Kohinoor Devanagari", sans-serif;
+  --font-family-noto-sans-ethiopic: "Noto Sans Ethiopic", Arial, Verdana, Geneva, Helvetica, sans-serif;
+  --font-family-noto-sans-gurmukhi: "Noto Sans Gurmukhi", Gautami, "Kohinoor Gurmukhi", sans-serif;
+  --font-family-noto-sans-sinhala: "Noto Serif Sinhala", Arial, Verdana, Geneva, Helvetica, sans-serif;
+  --font-family-noto-sans-tamil: "Noto Sans Tamil", Helmet, Freesans, Helvetica, Arial, sans-serif;
+  --font-family-padauk: Padauk, Arial, Verdana, Geneva, Helvetica, sans-serif;
+  --font-family-reith-sans: ReithSans, Helvetica, Arial, sans-serif;
+  --font-family-reith-serif: ReithSerif, Helvetica, Arial, sans-serif;
+  --font-family-reith-qalam: "BBC Reith Qalam", "Times New Roman", Arial, Verdana, Geneva, Helvetica, sans-serif;
+  --font-family-tahoma: Tahoma, Helmet, freesans, Helvetica, Arial, sans-serif;
```

Comment:
converted quite nicely to css properties: https://github.com/bbc/simorgh/blob/latest/src/app/components/ThemeProvider/fontFamilies.ts